packages/ember-application/lib/system/resolver.js

```diff
@@ -197,7 +197,7 @@ export default EmberObject.extend({
 
   parseName(fullName) {
     return this._parseNameCache[fullName] || (
-      (this._parseNameCache[fullName] = this._parseName(fullName))
```

Comment:
These extra parens are pretty common when doing an assignment inside of a boolean expression.